Core/include/Acts/Material/AccumulatedVolumeMaterial.hpp

```diff
@@ -35,7 +35,6 @@ class AccumulatedVolumeMaterial {
   double m_totalRho{0.};
   double m_thickness{1.};
   unsigned int m_materialEntries{0};
-  unsigned int m_vacuumEntries{0};
```

Comment:
Not used anywhere (anymore)?

Reply:
@Corentin-Allaire should comment.

Reply:
Yes my bad, it was used in the previous implementation of the AccumalatedVolumeMaterial and I forgot to remove it. It should definitely be not be there